src/js/app.js

```diff
@@ -0,0 +1,30 @@
+var newSearch;
+var searchResults;
+
+$(document).ready(function() {
+  // $('#testButton').on('click', function(e) {
+  //   e.preventDefault();
+  //   alert('Button was tested, gov\'nuh!');
+  // });
+  $('#searchSubmit').on('click', function(e) {
+    e.preventDefault();
+    newSearch = $('#searchInput').val();
```

Comment:
Since `newSearch` only exists inside this click listener, you can declare it right here, instead of line 1.